packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/import/vsphere-user-import.html

```diff
@@ -0,0 +1,108 @@
+<div
+    data-wizard
+    data-wizard-title=":: 'dedicatedCloud_USER_import_title' | translate"
+    data-wizard-on-cancel="$ctrl.goBack.bind($ctrl)"
+    data-wizard-on-finish="$ctrl.close.bind($ctrl)"
+    data-wizard-hide-cancel-button="currentStep !== 1"
+    data-wizard-hide-close-button="currentStep !== 1"
+    data-wizard-confirm-button-text=":: 'dedicatedCloud_USER_import_button_close' | translate"
+    data-wizard-cancel-button-text=":: 'dedicatedCloud_USER_import_button_close' | translate"
+    data-wizard-hide-previous-button="true"
+    data-wizard-keydown-disabled
+    data-wizard-bread-crumb
+>
+    <div data-wizard-step data-wizard-step-valid="$ctrl.formIsValid()">
+        <div class="text-center" data-ng-if="$ctrl.loaders.init">
+            <oui-spinner></oui-spinner>
+        </div>
+
+        <form data-ng-if="!$ctrl.loaders.init" name="importUserForm" novalidate>
+            <p data-translate="dedicatedCloud_USER_import_content"></p>
+
+            <oui-select
+                data-items="$ctrl.federationList"
+                data-match="ldapHostname"
+                data-model="$ctrl.model.domainName"
+                name="selectImportFederation"
+                placeholder="{{ :: 'dedicatedCloud_USER_import_select_federation_placeholder' | translate }}"
+                on-change="$ctrl.onFederationChange(modelValue)"
+                class="mt-4 mb-4"
+                required
+            >
+            </oui-select>
+
+            <oui-radio
+                model="$ctrl.model.type"
+                name="importUserRadio"
+                on-change="$ctrl.onImportTypeChange(modelValue)"
+                data-value="'user'"
+                class="mb-2"
+            >
+                <span data-translate="dedicatedCloud_USER_import_radio_user">
+                </span>
+            </oui-radio>
+
+            <oui-field
+                label="{{:: 'dedicatedCloud_USER_import_field_label_userName' | translate }} *"
+                data-ng-if="$ctrl.model.type === 'user'"
+            >
+                <input
+                    class="oui-input"
+                    name="userNameField"
+                    type="text"
+                    placeholder="{{ $ctrl.model.userExample.user }}"
+                    data-ng-model="$ctrl.model.userName"
+                    required
+                />
+            </oui-field>
+
+            <oui-radio
+                model="$ctrl.model.type"
+                name="importGroupRadio"
+                on-change="$ctrl.onImportTypeChange(modelValue)"
+                data-value="'group'"
```

Comment:
for consistency, use `data-` prefix or remove one `data-` for `data-value`.

Reply:
It's applied